Preferences.sublime-settings

```diff
@@ -8,5 +8,11 @@
 	"kickass_output_path": "bin",
 	"default_prebuild_path": "",
 	"default_postbuild_path": "",
-	"default_make_path": ""
+	"default_make_path": "",
+	"kickass_debug_command_c64debugger": "\"${kickass_debug_path}\" -autojmp -layout 10 -breakpoints \"${kickass_output_path}/${build_file_base_name}.dbg\" -symbols \"${kickass_output_path}/${build_file_base_name}.vs\" -wait 2500 -prg \"${kickass_output_path}/${build_file_base_name}.prg\"",
+	"kickass_debug_command_x64": "\"${kickass_debug_path}\" -logfile \"${kickass_output_path}/${build_file_base_name}_ViceLog.txt\" -moncommands \"${kickass_output_path}/${build_file_base_name}_MonCommands.mon\" ${kickass_debug_args} \"${kickass_output_path}/${build_file_base_name}.prg\"",
+	"kickass_run_command_x64": "\"${kickass_run_path}\" -logfile \"${kickass_output_path}/${build_file_base_name}_ViceLog.txt\" -moncommands \"${kickass_output_path}/${build_file_base_name}.vs\" ${kickass_run_args} \"${kickass_output_path}/${build_file_base_name}.prg\"",
+	"kickass_run_command_c64debugger": "\"${kickass_run_path}\" -autojmp -layout 1 -symbols \"${kickass_output_path}/${build_file_base_name}.vs\" -wait 2500 -prg \"${kickass_output_path}/${build_file_base_name}.prg\"",
+	"kickass_compile_parameters": "\"${build_file_base_name}.${file_extension}\" -log \"${kickass_output_path}/${build_file_base_name}_BuildLog.txt\" -o \"${kickass_output_path}/${build_file_base_name}.prg\" -vicesymbols -showmem -symbolfiledir \"${kickass_output_path}\" ${kickass_args}",
+	"kickass_compile_debug_additional_parameters": "-afo :afo=true :usebin=true",
```

Comment:
Rename to `kickass_compile_debug_additional_args` for consistency

Reply:
done